src/components/AddressSearch.js

```diff
@@ -36,51 +35,44 @@ const defaultProps = {
 };
 
 const AddressSearch = (props) => {
-    const googlePlacesRef = useRef();
-    useEffect(() => {
-        if (!googlePlacesRef.current) {
-            return;
-        }
-
-        googlePlacesRef.current.setAddressText(props.value);
-    }, []);
+    const [skippedFirstOnChangeText, setSkippedFirstOnChangeText] = useState(false);
 
     const saveLocationDetails = (details) => {
         const addressComponents = details.address_components;
-        if (isAddressValidForVBA(addressComponents)) {
```

Comment:
Why did we need this validation before, but not anymore?

Reply:
Because we are allowing the user to modify all fields manually even after selecting a result from the autocomplete.

If the user selects a result that is missing some parts of the address, we don't care really, we will just use the parts we got, and the user will have to manually input the rest. Before, if a result was incomplete, there was no way for the user to fill the missing information.

Reply:
Got it, thanks that makes sense to me.